MAC compile Error

```
core/math/expression.cpp:2117:124: error: conversion from 'long' to 'const Variant' is ambiguous
        ClassDB::bind_method(D_METHOD("execute", "inputs", "base_instance", "show_error"), &Expression::execute, DEFVAL(Array()), DEFVAL(NULL), DEFVAL(true));
                                                                                                                                  ^~~~~~~~~~~~
core/class_db.h:42:26: note: expanded from macro 'DEFVAL'
#define DEFVAL(m_defval) (m_defval)
                         ^~~~~~~~~~
core/variant.h:233:2: note: candidate constructor
        Variant(bool p_bool);
        ^
core/variant.h:234:2: note: candidate constructor
        Variant(signed int p_int); // real one
        ^
core/variant.h:235:2: note: candidate constructor
        Variant(unsigned int p_int);
        ^
core/variant.h:241:2: note: candidate constructor
        Variant(signed short p_short); // real one
        ^
core/variant.h:242:2: note: candidate constructor
        Variant(unsigned short p_short);
        ^
core/variant.h:243:2: note: candidate constructor
        Variant(signed char p_char); // real one
        ^
core/variant.h:244:2: note: candidate constructor
        Variant(unsigned char p_char);
        ^
core/variant.h:245:2: note: candidate constructor
        Variant(int64_t p_char); // real one
        ^
core/variant.h:246:2: note: candidate constructor
        Variant(uint64_t p_char);
        ^
core/variant.h:247:2: note: candidate constructor
        Variant(float p_float);
        ^
core/variant.h:248:2: note: candidate constructor
        Variant(double p_double);
        ^
core/variant.h:251:2: note: candidate constructor
        Variant(const char *const p_cstring);
        ^
core/variant.h:252:2: note: candidate constructor
        Variant(const CharType *p_wstring);
        ^
core/variant.h:266:2: note: candidate constructor
        Variant(const Object *p_object);
```

--- a/core/math/expression.cpp
+++ b/core/math/expression.cpp
@@ -2114,7 +2114,7 @@ String Expression::get_error_text() const {
 void Expression::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("parse", "expression", "input_names"), &Expression::parse, DEFVAL(Vector<String>()));
-	ClassDB::bind_method(D_METHOD("execute", "inputs", "base_instance", "show_error"), &Expression::execute, DEFVAL(Array()), DEFVAL(NULL), DEFVAL(true));
+	ClassDB::bind_method(D_METHOD("execute", "inputs", "base_instance", "show_error"), &Expression::execute, DEFVAL(Array()), DEFVAL(Variant()), DEFVAL(true));
 	ClassDB::bind_method(D_METHOD("has_execute_failed"), &Expression::has_execute_failed);
 	ClassDB::bind_method(D_METHOD("get_error_text"), &Expression::get_error_text);
 }

